index.html

```diff
@@ -54,63 +55,76 @@ <h3>Сувениры</h3>
-              <span>-№4-</span>
-              <p>Ехать в Седону из Лас-Вегаса совсем <br>не скучно!</p>
-           </li>
-         </div>
-          <div class="another-reason">
-            <li>
-              <h3>Мало <br> туристов</h3>
-              <span>-№5-</span>
-              <p>Большинство едет в Гранд Каньон <br>и толпится там</p>
-            </li>
-          </div>
+          </li>
+          <li class="another-reason">
+            <h3>Красивая <br> дорога</h3>
+            <span>-№4-</span>
+            <p>Ехать в Седону из Лас-Вегаса совсем <br>не скучно!</p>
+          </li>
+          <li class="another-reason">
+            <h3>Мало <br> туристов</h3>
+            <span>-№5-</span>
+            <p>Большинство едет в Гранд Каньон <br>и толпится там</p>
+          </li>
         </ol>
       </section>
       <section>
         <div class="interested">
           <h3>Заинтересовались?</h3>
           <p>Укажите предполагаемые даты поездки, <br> и мы покажем вам лучшие предложения гостиниц в Седоне</p>
         </div>
-        <button type="button" class="button button-subtitle">Поиск гостиницы в Седоне</button>
-        <form action="https://echo.htmlacademy.ru" method="GET" class="search-form">
-          <p>
-            <label class="form">Дата заезда:<input type="text" name="date-enter" placeholder="24 апреля 2017"></label>
-            <button type="button"><img src="img/calendar.svg" alt="календарь"></button>
-          </p>
-          <p>
-            <label class="form">Дата выезда:<input type="text" name="date-leave" placeholder="4 июля 2017"></label>
-            <button type="button"><img src="img/calendar.svg" alt="календарь"></button>
-          </p>
-          <p>
-            <button type="button"><img src="img/minus.svg" alt="-"></button>
-            <label class="form">Взрослые:<input type="text" name="adults" placeholder="2"></label>
-            <button type="button"><img src="img/plus.svg" alt="+"></button>
-          </p>
-          <p>
-            <button type="button"><img src="img/minus.svg" alt="-"></button>
-            <label class="form">Дети:<input type="text" name="children" placeholder="0"></label>
-            <button type="button"><img src="img/plus.svg" alt="+"></button>
-          </p>
-          <button type="submit" class="button button-search">Найти</button>
-        </form>
+        <div class="main-form">
+          <button type="button" class="button button-subtitle">Поиск гостиницы в Седоне</button>
+          <form action="https://echo.htmlacademy.ru" method="GET" class="search-form">
+            <p class="date-form">
+              <label class="form" for="date-enter">Дата заезда:</label>
+              <div class="date">
+                <input type="text" name="date-enter" placeholder="24 апреля 2017" id="date-enter">
+                <button type="button"><img src="img/calendar.svg" alt="календарь"></button>
+              </div>
+            </p>
```

Comment:
Параграф может содержать только строчные элементы. `div` является блочным, поэтому нужно изменить разметку. `div` и `p` поменять местами:
```suggestion
            <div class="date-form">
              <label class="form" for="date-enter">Дата заезда:</label>
              <p class="date">
                <input type="text" name="date-enter" placeholder="24 апреля 2017" id="date-enter">
                <button type="button"><img src="img/calendar.svg" alt="календарь"></button>
              </p>
            </div>
```